test-harness/specs/default_responses-AC-1.oas3.txt

```diff
@@ -0,0 +1,26 @@
+====test====
+When I send a request to an operation
+And this operation is defined
+And this operation has only a default response defined
+Then I should get back the default response
+====spec====
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      responses:
+        default:
+          description: default response
+          content:
+            application/json:
+              example: test
+====server====
+mock -p 4010
+====command====
+curl -i -X GET http://localhost:4010/todos
```

Comment:
Out of interest, why do we always have -i when we dont actually care about response headers? Sometimes we care about headers, but sometimes we dont, should we maybe remove it from tests where we dont care?

Reply:
Without the `-i` flag the library we're using to parse and compare the output (http-string-parser) fails. It's outlined in the README of the harness

Reply:
The question [here](https://github.com/stoplightio/prism/pull/465#discussion_r304365627) also applies here 

We talked about having one gavel based expect and one literal string match. We wouldn’t need to http parse if it’s a literal string match. Remember that not all expect will be an entire HTTP Response, because not every single command is going to be a pure curl request.

Reply:
I think I was wrong and we can simply *not* use the `-i` flag

Reply:
Is prism actually returning that literally, or was this written in incorrectly and gavel is comparing the two things and evaluating as true? 

I am asking how we can have literal expect and gavel based expect because I want o use literal expect in most instances to avoid this sort of confusion.

Reply:
Prism is literally returning that value — I'd say it's a bug indeed, since such header (I think) is not valid.

I've fixed it in 974aab0 — but if so we might need to revisit some other tests here because two of them are really expecting */* as content type

Reply:
> I am asking how we can have literal expect and gavel based expect because I want o use literal expect in most instances to avoid this sort of confusion.

`expect` = gavel validation + body matching (key and value)
`expect-loose`= gavel validation only

Reply:
Oh ok, thats a bit different from what I thought. I thought there was expect = strict and expect-loose would be gavel. Nevermind, another topic for another time. 

No test should expect `*/*` so lets delete them or change them.

Reply:
They've been fixed — the idea is that if we have such situation we return text/plain